package.json

```diff
@@ -14,8 +14,10 @@
         "@next/font": "13.1.6",
         "eslint-config-next": "13.1.6",
         "next": "13.1.6",
+        "prettier": "^2.8.4",
         "react": "18.2.0",
-        "react-dom": "18.2.0"
+        "react-dom": "18.2.0",
+        "react-modal": "^3.16.1"
```

Comment:
Any additional dependency should be discussed with the team. Please create a modal component :)

Reply:
Modal created inside Form component and react-modal removed from project dependencies